packages/rum-core/src/opentracing/tracer.js

```diff
@@ -74,7 +74,7 @@ class Tracer extends otTracer {
     var span
     var currentTransaction = this.transactionService.getCurrentTransaction()
 
-    if (currentTransaction && !currentTransaction.ended) {
+    if (currentTransaction) {
```

Comment:
Do we need to create spans if there is no active transaction?

Reply:
I think this was part of [this refactoring](https://github.com/elastic/apm-agent-rum-js/pull/259/files#diff-074367311a4235f7774f1d58a437e7b6R66). I will move it there to make it clear.